src/soundio/soundmanagerios.mm

```diff
@@ -0,0 +1,31 @@
+#include <QDebug>
+#include <QtGlobal>
+
+#import <AVFAudio/AVFAudio.h>
+
+namespace mixxx {
+
+void initializeAVAudioSession() {
+    AVAudioSession* session = AVAudioSession.sharedInstance;
+    AVAudioSessionCategory category = AVAudioSessionCategoryPlayback;
+    AVAudioSessionMode mode = AVAudioSessionModeDefault;
+    AVAudioSessionCategoryOptions options =
+            AVAudioSessionCategoryOptionMixWithOthers |
+            AVAudioSessionCategoryOptionAllowAirPlay |
+            AVAudioSessionCategoryOptionAllowBluetoothA2DP;
+
+    NSError* error = nil;
```

Comment:
What I have read is: nil is defined as (id)0. I think we should use NULL here in objective C.

Reply:
Or is the pointer asterisk wrong? Because of & below?

Reply:
By convention, `nil` is the preferred spelling for Objective-C objects. And the double pointer is intended, as every Objective-C object has to be a pointer, but the API additionally needs to reassign the value here (i.e. it is an output parameter)

Reply:
Thank you for explaining. I cross read a bit about the topic and I think I got it.